frontend/src/app/shared/components/cluster-table/renderer/cluster-actions-renderer.component.ts

```diff
@@ -25,8 +25,8 @@ import { Cluster } from "src/app/modules/cluster/models/cluster";
       <div class="flex">
           <button
                 routerLink="/clusters/detail/{{ data.id }}"
-                class="btn">
-            <svg-icon src="assets/icons/outline/arrow-sm-right.svg" [svgClass]="'h-5 w-5'"> </svg-icon>
+                class="bg-blue-500 hover:bg-blue-600 text-white font-medium py-2 px-4 rounded text-sm transition-colors">
```

Comment:
We can add to [here](https://github.com/ArubaKube/liqo-dashboard/blob/c7a99caec22aac6c671d0dc3c17d02895478f644/frontend/src/styles.scss#L77) in the style.scss a new class, so that we can reuse it:
```
	.btn-primary {
		@apply bg-blue-500 hover:bg-blue-600 text-white;
	}
```

then we can directly apply the class to the button:
```suggestion
                class="btn btn-primary">
```